src/presentation/public/manifest.json

```diff
@@ -0,0 +1,62 @@
+{
+    "name": "privacy.sexy",
+    "short_name": "privacy.sexy",
+    "description": "Enforce privacy & security best-practices on Windows, macOS and Linux, because privacy is sexy.",
+    "version": "0.13.5",
+    "start_url": "/",
+    "display": "standalone",
+    "background_color": "#ffffff",
+    "theme_color": "#000000",
+    "icons": [
```

Comment:
We should be able to automatically update the icons.

There's [`logo-update.js`](https://github.com/undergroundwires/privacy.sexy/blob/master/scripts/logo-update.js) which is executed when you run `npm run build:icons`. It creates all logos from single `logo.svg` , see [`README.md`](https://github.com/undergroundwires/privacy.sexy/blob/master/img/README.md). We should update the script so it generates all needed icons in the icons folder (populates from the SVG).

This helps maintaining the project by updating the logo in single file/place and knowing that it will be updated throughout the application after single command.

Reply:
Hello @undergroundwires,

I attempted to run the icons:build command, but it is dependent on ImageMagick. The software uses the convert command, which conflicts with the convert.exe utility in Windows. The convert.exe file in Windows is a command-line utility used to convert file systems on a volume to FAT32 or NTFS. Due to this conflict, I am encountering several errors.